### PR TITLE
rhche.openshift.io is not valid URL anymore.

### PR DESCRIPTION
**Pull request by rhopp**
Signed-off-by: Radim Hopp <rhopp@redhat.com>

### CODE REVIEW

**Comment by rhopp** [comment]
[test]

**Comment by rhopp** [comment]
[test]

**Comment by rhopp** [comment]
I suspect this may be the reason of tests failure of PR #333 . I'll go ahead and merge this one, rebase #333 and see... Any objections @Katka92 @ScrewTSW ?

File: che-start-workspace/_setenv.sh [hunk 1]
```diff
@@ -13,8 +13,7 @@
 #export AUTH_SERVER_URL=https://auth.openshift.io
 
 # Che server url.
-#export CHE_SERVER_URL=https://che.openshift.io
-export CHE_SERVER_URL=https://rhche.openshift.io
+export CHE_SERVER_URL=https://che.openshift.io
```
Comment (Katka92) [inline che-start-workspace/_setenv.sh]: As the value is same here and in job definition - not sure if we need this line uncommented.

**Pull request merged**